### ISSUE TITLE
Chapter 6 Exercises 8 Grow a forest 'continuous-multioutput'

### ISSUE THREAD

**Issue by Ilya-Curie** [ageron/handson-ml2#292]
# Context: Exercise 7 

**a**
<code>
from sklearn.datasets import make_moons

X, y = make_moons(n_samples=10000, noise=0.4, random_state=42)
</code>

**b**
<code>
from sklearn.model_selection import train_test_split
</code>
<code>
x_train, x_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
</code>

**c**
<code>
from sklearn.model_selection import GridSearchCV
</code>
<code>
from sklearn.tree import DecisionTreeClassifier
</code>

<code>
params={"max_leaf_nodes":list(range(0,100)), "min_samples_split":[2,3,4,5]}

tree_clf=DecisionTreeClassifier(random_state=42)
</code>
<code>
model = GridSearchCV(tree_clf, params, cv=3, verbose=1)
model.fit(x_train, y_train)
</code>

# Code Exercise 8

**a**
<code>
from sklearn.model_selection import ShuffleSplit
</code>
<code>
n_trees = 1000
n_instances = 100
</code>
<code>
mini_sets = []
</code>
<code>
rs = ShuffleSplit(n_splits=n_trees, test_size=len(x_train) - n_instances, random_state=42)
</code>
<code>
for mini_train_index, mini_test_index in rs.split(x_train):
</code>
<code>
x_mini_train = x_train[mini_train_index]
</code>
<code>
y_mini_train = x_train[mini_train_index]
</code>
<code>
mini_sets.append((x_mini_train, y_mini_train))
</code>

**b**
<code>
from sklearn.base import clone
</code>
<code>
forest = [clone(model.best_estimator_) for _ in range(n_trees)]
</code>
<code>
accuracy_scores = []
</code>
<code>
for tree, (x_mini_train, y_mini_train) in zip(forest, mini_sets):
</code>
<code>    
tree.fit(x_mini_train, y_mini_train)
</code>
<code>
y_pred = tree.predict(x_test)
</code>
<code>
accuracy_scores.append(accuracy_score(y_test, y_pred))
</code>
<code>
np.mean(accuracy_scores)
</code>

## Obtained error
*----> 8     tree.fit(x_mini_train, y_mini_train)*
...
**ValueError:** Unknown label type: **'continuous-multioutput'**

## How to fixe it and why does it happen?

Thanks!
